doc/superbuild/source/developers/testing/Answers.rst

```diff
@@ -0,0 +1,118 @@
+
+Sometimes it is necessary to update the answer files, or add a new file, so that the tests pass, if a change in output
+is expected/desired. Changing output files requires careful verification that the output is still
+"correct". The procedure for updating answer files is as follows:
+
+- For the ``.out`` files embedded in examples, simply overwrite the file with the new output. If the
+  tests still fail, then you will need to download the output file generated on the Jenkins CI
+  machine and commit it. Extracting this file from the Jenkins machine may require help from David
+  or Cody currently.
+
+- For the ``.out`` files in `sundials-codes/answers <https://github.com/sundials-codes/answers>`_:
+
+  #. Navigate to `test/answers` in the sundials repository. If there is nothing in this directory,
+     you need to run `git submodule init && git submodule update`. Alternatively clone the answers
+     repository outside of the SUNDIALS source tree. 
+
+  #. Checkout the main branch: ``git checkout main``. 
+
+  #. Create a new branch off of main. It is ideal if you use a branch name that matches the name of
+     the branch that you are developing in sundials on: ``git checkout -b <branch name>``.
+
+  #. Update the relevant ``.out`` files in
+     ``test/answers/linux-ubuntu20.04-x86_64/gcc-9.4.0/<double|single|extended>`` (this is the files
+     used for the GitHub Actions CI). Like with the embedded `.out` files, you can try and use the
+     output generated on your machine, but it may be different enough from what is produced on the
+     GitHub actions to trigger a failure. You can download the output files generated on the GitHub
+     machines by going to `<https://github.com/LLNL/sundials/actions>`_ finding your failing
+     test, clicking it, then at the bottom downloading the "output_files" artifact (you can also find your
+     failing test at the bottom of a PR). The downloaded zip file will be the SUNDIALS build
+     directory. To update just the failed files you can use ``scripts/updateOutFiles.py ``e.g.,
+    
+     .. code::
+
+         cd scripts  
+         ./updateOutFiles.py <source path> <destination path>  
+      
+     When updating files in the answers repo the source path is the path to the build directory
+     downloaded from GitHub and the destination path is ``<path to answers
+     repo>/linux-ubuntu20.04-x86_64/gcc-9.4.0/<double|single|extended>``. When updating with output
+     from Jenkins the source path is the build directory for the Jenkins run and the destination
+     path is the top level of the sundials repo (i.e., ../. if running from scripts). The script
+     will automatically traverse the examples and test/unit_test directories to update output files
+     for failed tests.
+
+  #. If you need to use ssh authentication for pushing to GitHub, then you may need to update the 
+     remote for the submodule:
+
+   .. code::
+
+         git remote set-url origin git@github.com:LLNL/sundials.git
```

Comment:
Is this the right remote? Or should it be sundials-codes/answers

Reply:
Good catch, it should be sundials-codes/answers.

Reply:
```suggestion
         git remote set-url origin git@github.com:sundials-codes/answers.git
```